hedera-mirror-common/src/test/java/com/hedera/mirror/common/domain/transaction/BlockItemTest.java

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright (C) 2025 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.mirror.common.domain.transaction;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.hedera.hapi.block.stream.output.protoc.TransactionResult;
+import com.hederahashgraph.api.proto.java.ResponseCodeEnum;
+import com.hederahashgraph.api.proto.java.Timestamp;
+import com.hederahashgraph.api.proto.java.Transaction;
+import java.util.List;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class BlockItemTest {
+
+    static Stream<ResponseCodeEnum> validStatuses() {
+        return Stream.of(
+                ResponseCodeEnum.FEE_SCHEDULE_FILE_PART_UPLOADED,
+                ResponseCodeEnum.SUCCESS,
+                ResponseCodeEnum.SUCCESS_BUT_MISSING_EXPECTED_OPERATION);
+    }
+
+    @ParameterizedTest
+    @MethodSource("validStatuses")
+    void parseSuccess_whenNoParentPresentAndCorrectStatus_returnTrue(ResponseCodeEnum status) {
```

Comment:
We don't use this underscore syntax. You should use `@DisplayName` if you want more verbose test names with variable replacement.
```suggestion
    @DisplayName("isSuccessful() returns true")
    @EnumSource(value = ResponseCodeEnum.class, mode = INCLUDE, names = "SUCCESS,...")
    @ParameterizedTest("with status {0}")
    void isSuccessfulWithSuccessEnums(ResponseCodeEnum status) {
```